src/dmd/mars.d

```diff
@@ -1386,16 +1467,42 @@ bool parseCommandLine(const ref Strings arguments, const size_t argc, ref Param
             params.useDeprecated = Diagnostic.inform;
         else if (arg == "-c")                // https://dlang.org/dmd.html#switch-c
             params.link = false;
-        else if (startsWith(p + 1, "check=")) // https://dlang.org/dmd.html#switch-check
+        else if (startsWith(p + 1, "checkaction")) // https://dlang.org/dmd.html#switch-checkaction
+        {
+            /* Parse:
+             *    -checkaction=D|C|halt
+             */
+            enum l = "-checkaction=".length;
```

Comment:
Don't use `l` for variable names because it looks like `1`. Use `len` or `length`.